packages/firebase_dynamic_links/.gitignore

```diff
@@ -0,0 +1,8 @@
+.DS_Store
```

Comment:
Please remove all `.gitignore` files from individual plugins. If you rebase to master, the repo root `.gitignore` should make git ignore all the files we don't want to check in.